puppetmaster/applications/microk8s.py

```diff
@@ -0,0 +1,34 @@
+from subprocess import check_output
+
+from puppetmaster.applications import BaseApplication, BaseProvider, Event, Events
+
+
+class MicroK8sApplication(BaseApplication):
+    """
+    MicroK8s Application.
+    """
+
+    def __init__(self):
+        raise NotImplementedError("Must use a real application.")
```

Comment:
Sick burn, but I'm guessing this is a copy-pasta error.

Reply:
haha, sadly not, it should say "application instance" I guess, but it's going away with the decorator (I think).